### PR TITLE
Add anti-captcha support

### PR DESCRIPTION
**Pull request by jmslagle**
- NEEDS TESTED but it should be fine.  Has new requirements.txt item

### CODE REVIEW

File: monocle/worker.py [hunk 1]
```diff
@@ -1481,59 +1484,73 @@ def smart_throttle(self, requests=1):
...
+                raise CaptchaSolveException from e
+
+            token = response.get('request')
+            if not response.get('status') == 1:
+                self.log.error("Failed to get CAPTCHA response: {}", token)
+                raise CaptchaSolveException
+        else:
+            try:
+                acclient = AnticaptchaClient(conf.CAPTCHA_KEY)
+                actask = NoCaptchaTaskProxylessTask(challenge_url, '6LeeTScTAAAAADqvhqVMhPpr_vB9D364Ia-1dSgK')
```
Comment (MonkeyTedBaker) [inline monocle/worker.py]: What is throw scaring looking key? Is it the same for everyone?

Comment (captainmedic) [reply monocle/worker.py]: Are you referring to the site key?

Comment (MonkeyTedBaker) [reply monocle/worker.py]: 6LeeTScTAAAAADqvhqVMhPpr...

Comment (captainmedic) [reply monocle/worker.py]: Yea that is the dsk for pokemon go captcha

File: config.example.py [hunk 2]
```diff
@@ -260,6 +260,8 @@
 #CAPTCHAS_ALLOWED = 3
 ## Get new accounts from the CAPTCHA queue first if it's not empty
 #FAVOR_CAPTCHA = True
+## Use anticaptcha instead of 2captcha
+#USE_ANTICAPTCHA = True
```
Comment (MonkeyTedBaker) [inline config.example.py]: This needs to be defined and set default False in sanitized.py

Comment (jmslagle) [reply config.example.py]: Done :)

File: monocle/worker.py (hunk 1 above)
Comment (jmslagle) [reply monocle/worker.py]: Yup - this is the same domain key that 2captcha is solving against.

**Comment by MonkeyTedBaker** [comment]
Thanks. Now let’s wait for some fish to test this out. Lol

**Comment by beloware** [comment]
Fis is here volunteer for hooking :D

**Comment by MonkeyTedBaker** [comment]
Tested 2captcha did not break. So merging.

**Pull request merged**